sklearn/preprocessing/_discretization.py

```diff
@@ -115,10 +118,15 @@ class KBinsDiscretizer(TransformerMixin, BaseEstimator):
            [ 0.5,  3.5, -1.5,  1.5]])
     """
 
-    def __init__(self, n_bins=5, encode='onehot', strategy='quantile'):
+    def __init__(self,
+                 n_bins=5,
+                 encode='onehot',
+                 strategy='quantile',
+                 dtype='float64'):
```

Comment:
Currently we mostly use the following style,
```

    def __init__(self, n_bins=5, encode='onehot', strategy='quantile',
                 dtype=np.float64):
```

Reply:
Done! Changed the strings to numpy objects